discovery/kubernetes/kubernetes.go

```diff
@@ -16,6 +16,7 @@ package kubernetes
 import (
 	"context"
 	"fmt"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
That import is not at the correct location

Reply:
> That import is not at the correct location

So, what is the correct one?

Reply:
> That import is not at the correct location

but so, why the CI succeeds if a lint check should fail this?

Reply:
> That import is not at the correct location

BTW, I've fixed the import section